tests/test_basics.py

```diff
@@ -158,12 +143,11 @@ def test_astropy_time_from_tai_unix(self) -> None:
         for dt in (-1, -0.5, -0.1, 0, 0.1, 1):
             with self.subTest(dt=dt):
                 utc_unix = unix_time0 + dt
-                tai_unix = salobj.tai_from_utc(utc_unix)
-                astropy_time1 = salobj.astropy_time_from_tai_unix(tai_unix)
-                self.assertIsInstance(astropy_time1, astropy.time.Time)
-                self.assertEqual(astropy_time1.scale, "tai")
-                tai_unix_round_trip1 = salobj.tai_from_utc(astropy_time1)
-                self.assertAlmostEqual(tai_unix, tai_unix_round_trip1, delta=1e-6)
+                tai_unix = utils.tai_from_utc(utc_unix)
+                with self.assertWarns(DeprecationWarning):
```

Comment:
Maybe add `# TODO: (DM-31660) Remove this deprecated wrapper`.

I also noticed that the other comments in `bade.py` do not mark it as `TODO`. No big deal but you may want to add "TODO".

Reply:
I changed them all to `# TODO DM-31660: